frontendActions/src/main/resources/GitMacheteBundle.properties

```diff
@@ -0,0 +1,222 @@
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.disabled.repository.unknown-state=Cannot rebase due to unknown repository state
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.disabled.root-branch=Root branch ''{0}'' cannot be rebased
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.repository.state.detached-head=in the detached head state
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.repository.state.ongoing.cherry-pick=during an ongoing cherry-pick
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.repository.state.ongoing.merge=during an ongoing merge
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.repository.state.ongoing.rebase=during an ongoing rebase
+action.GitMachete.BaseRebaseBranchOntoParentAction.description.repository.state.ongoing.revert=during an ongoing revert
+action.GitMachete.BaseRebaseBranchOntoParentAction.hook.task-title=Running machete-pre-rebase hook
+action.GitMachete.BaseRebaseBranchOntoParentAction.notification.abort=Rebase aborted
+action.GitMachete.BaseRebaseBranchOntoParentAction.notification.fail=Rebase failed
+action.GitMachete.BaseRebaseBranchOntoParentAction.task-title=Rebasing
+action.GitMachete.BaseRebaseBranchOntoParentAction.text=_Rebase Branch onto Parent
+
+
+# used indirectly within BaseResetBranchToRemoteAction
+action.GitMachete.BaseResetBranchToRemoteAction.description.enabled={0} branch ''{1}'' to its remote tracking branch
+action.GitMachete.BaseResetBranchToRemoteAction.action-name=Re_set to Remote
+action.GitMachete.BaseResetBranchToRemoteAction.description-action-name=Reset to remote
+action.GitMachete.BaseResetBranchToRemoteAction.notification.success=Branch ''{0}'' reset to remote
+action.GitMachete.BaseResetBranchToRemoteAction.notification.title=Resetting
+action.GitMachete.BaseResetBranchToRemoteAction.task-title=Resetting...
+
+
+action.GitMachete.BaseSlideInBranchBelowAction.description=Slide in branch below ''{0}''
+action.GitMachete.BaseSlideInBranchBelowAction.description.disabled.no-parent=Slide in disabled due to undefined parent branch
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.create-new-branch.title=Create New Branch Below {0}
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.label=Branch name:
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.label.parent=Parent Branch:
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.ok-button=Slide In
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.title=Slide In Branch Below
+action.GitMachete.BaseSlideInBranchBelowAction.notification.fail=Slide in of branch <b>{0}</b> failed
+action.GitMachete.BaseSlideInBranchBelowAction.notification.fail.branch-layout-write=New branch layout write failed
+action.GitMachete.BaseSlideInBranchBelowAction.notification.message.branch-name-equals-parent=Cannot slide in the branch under itself
```

Comment:
I don't think `branch-name-equals-parent` and `Cannot slide in the branch under itself` refer to the same case